robot.py

```diff
@@ -90,10 +90,18 @@ def arm_movement(self, speed=200, millies=0):
     
     
     def gyro_angle(self, angle):
-        self.brake()
+        ''' Turns to desired angle
+
+        Args: 
+        angle: the angle you want the robot to turn
+        '''
+        self.gyro.reset_angle(0)
```

Comment:
@Bir9 can you please remind me why we want to reset the angle here?

Reply:
Let's create a replacement PR in which we avoid this. Also please keep the old print messages (i.e., with a prefix message).